src/protocolgame.cpp

```diff
@@ -2390,6 +2404,9 @@ void ProtocolGame::sendAddCreature(const Creature* creature, const Position& pos
 	msg.addByte(0x00); // can change pvp framing option
 	msg.addByte(0x00); // expert mode button enabled
 
+	msg.addString("http://static.tibia.com/images/store/");
+	msg.addByte(g_config.getNumber(ConfigManager::MAX_PACKETS_PER_SECOND));
```

Comment:
Im not fully gonna take credit for these fails, but when I started to compare the changes and noticed 25 and rememberd the max package config value.
And I wrote abt it in the 1090 thread....
But anyways do as summ said and create a new config integer.